src/modules/ecdsa_adaptor/main_impl.h

```diff
@@ -7,8 +7,8 @@
 #ifndef SECP256K1_MODULE_ECDSA_ADAPTOR_MAIN_H
 #define SECP256K1_MODULE_ECDSA_ADAPTOR_MAIN_H
 
-#include "include/secp256k1_ecdsa_adaptor.h"
-#include "modules/ecdsa_adaptor/dleq_impl.h"
+#include "../../../include/secp256k1_ecdsa_adaptor.h"
+#include "./dleq_impl.h"
```

Comment:
nit:

```suggestion
#include "dleq_impl.h"
```
(and many other lines)

For consistency with upstream secp256k1, which doesn't have the "./" but I'm happy to ACK either way.

Reply:
Ok, cool, I'll remove the `./`s (in my local testing they don't seem to matter)